### ISSUE TITLE
Update README to document development/build and prereqs

### ISSUE THREAD

**Issue by aaron-em** [firemacs/FiremacsWE#12]
Leftover from #4.

**Comment by kazu-yamamoto** [comment]
Yes. I would like to know how to use FiremacsWE without registration.

**Comment by brudgers** [comment]
@kazu-yamamoto Once you have downloaded the code, the extension can be loaded using http://about:debugging See, https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Your_first_WebExtension#Installing
